script/pageload/functions.js

```diff
@@ -9,27 +9,140 @@ var setThumbnail = function(text){
     chrome.extension.sendMessage(text);
 };
 
+var insertAfter = function(newNode, referenceNode) {
+    referenceNode.parentNode.insertBefore(newNode, referenceNode.nextSibling);
+}
+
+getAllBounties = function(){
+  var bounties_api_url = 'https://gitcoin.co/api/v0.1/bounties/?order_by=-web3_created';
```

Comment:
if you wanted to be more performant/selective, you could search bounties by keyword here (so that you don't have to match inside the for loop on line 28).

Here's an example: https://gitcoin.co/api/v0.1/bounties/?raw_data=python

Reply:
itd also be good to only show mainnet bounties that are open

https://gitcoin.co/api/v0.1/bounties/?&idx_status=open&network=mainnet

Reply:
Good idea, thanks for this

Reply:
mind making this change 

> itd also be good to only show mainnet bounties that are open

> https://gitcoin.co/api/v0.1/bounties/?&idx_status=open&network=mainnet